apps/docs/src/.vitepress/config.ts

```diff
@@ -35,16 +33,21 @@ export default defineConfig({
     },
     lastUpdated: {}, // needed to show the last updated text with default settings
     nav: [
-      { text: "Brand", link: "/brand/team", activeMatch: "/brand/" },
-      { text: "Basics", link: "/basics/", activeMatch: "/basics/" },
-      { text: "Tokens", link: "/tokens/introduction", activeMatch: "/tokens/" },
-      { text: "Development", link: "/development/", activeMatch: "/development/" },
+      {
+        text: "Docs",
+        items: [
+          { text: "Basics", link: "/basics/", activeMatch: "/basics/" },
+          { text: "Tokens", link: "/tokens/colors", activeMatch: "/tokens/" },
+          { text: "Development", link: "/development/", activeMatch: "/development/" },
+        ],
+      },
+      { text: "Components", link: "https://storybook.onyx.schwarz" },
+      { text: "Icons", link: "/icons" },
       {
         text: "Resources",
-        activeMatch: "/resources/",
         items: [
-          { text: "Icons", link: "/resources/icons" },
-          { text: "Storybook", link: storybookLink },
+          { text: "About", link: "/brand/team", activeMatch: "/brand/" },
```

Comment:
could you pls also rename "brand" here? :)
![image](https://github.com/SchwarzIT/onyx/assets/151019977/b723b567-7b02-4dcf-8338-b7f27668b181)